src/altair/state_transition/helpers.rs

```diff
@@ -117,7 +117,12 @@ pub fn compute_proposer_index<
     loop {
         let shuffled_index = compute_shuffled_index(i % total, total, seed, context)?;
         let candidate_index = indices[shuffled_index];
+        #[cfg(not(feature = "serde"))]
+        let i_bytes: [u8; 4] = (i / 32).to_le_bytes();
```

Comment:
I'm curious what is the purpose of this? It seems to be broken at the moment if the "serde" feature is not enabled

Reply:
@dharjeezy why is there a serde feature flag here?

Reply:
So, our target compilation is `wasm32-unknown-unknown`
I had to include the feature flag because of that else it errors out

@Wizdave97 @willemolding

Reply:
Collecting into a dynamic type like a Vec might work better in this case.

Reply:
Maybe better to use `#[cfg(target_arch = "wasm32")]` if it is a wasm specific thing? As it stands it errors our building for x86 with serde disabled

Reply:
Oh wait an even better solution is to remove the intermediate variable entirely e.g.
```rust
hash_input[32..].copy_from_slice(&((i / 32) as u64).to_le_bytes());
```
This will build either way regardless of the size of a u64 on the given target